asab/api/service.py

```diff
@@ -133,8 +133,11 @@ def initialize_zookeeper(self, zoocontainer=None):
 			zksvc = self.App.get_service("asab.ZooKeeperService")
 			zoocontainer = zksvc.DefaultContainer
 
-		# get zookeeper-serivice
+		# get zookeeper-service
 		self.ZkContainer = zoocontainer
+		taskservice = self.App.get_service("asab.TaskService")
+		zookeeperpath = asab.Config.get("asab:zookeeper", "path")
+		taskservice.schedule(self.ZkContainer.ZooKeeper.ensure_path(zookeeperpath + "/run"))
```

Comment:
This could lead to race condition b/c of the task service.

I believe that error should be handled here:
https://github.com/TeskaLabs/asab/blob/faad40a71e79aae12e22cb24060e875148e02f79/asab/zookeeper/container.py#L97

Reply:
:open_mouth: That's what you refused in the first place! :see_no_evil:

Reply:
Yeah, but this time you need to ensure `self.Path`, not `/run` ;-)

Reply:
Your second choice is also ok, but you need to do that synchronously, which is the problem.

Reply:
:bulb: